index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Africa Agriculture Summit|2023</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <script src="./js/index.js" defer></script>
+</head>
+
+<body>
+  <header>
+    <div class="desktop-nav">
+      <div class="face-twitt">
+        <img src="./images/facebook-512.png" alt="facebook" class="facebook">
+        <img src="./images/twitter-512.png" alt="twitter" class="twitter">
+      </div>
+      <ul class="english">
+        <li><a href="#">English</a></li>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    </div>
+    <div class="desktop-items">
+      <a href="index.html"><img src="./images/LogoMakr-45qqgi.png" alt="" class="desktop-logo"></a>
+      <ul>
+        <li><a href="about.html" class="about">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Sponsors</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#" class="campaign">AA Campaign</a></li>
+      </ul>
+    </div>
+    <nav class="hidden">
+      <img src="./images/cancel.png" alt="" class="cancel">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Feature</a></li>
+      </ul>
+    </nav>
+    <main class="main-wrapper">
+      <img src="./images/menu.png" class="menu" alt="menu-icon">
+      <div class="h1-cont">
+        <h1 class="title">AFRICA COMMONS AGRICULTURE SUMMIT 2023</h1>
+      </div>
+      <p class="motto">
+        Bringing together farmers, suppliers, processors, brands, academia,
+        conservation organizations and the public sector.
+      </p>
+      <p class="date">23 Dec 2023</p>
+      <p class="venue">Lagos Continental Hotel, Lag, Nig.</p>
+    </main>
+  </header>
+  <main class="">
+    <section class="main-program">
```

Comment:
- [Optional] On the home page, the main program section doesn't display completely, kindly fix this please.
![image](https://user-images.githubusercontent.com/94551484/227715799-97f91a6e-9b47-43ca-ba64-48364ea75dee.png)